dotcom-rendering/src/components/Titlepiece.tsx

```diff
@@ -132,124 +92,61 @@ const burgerStyles = css`
 		align-self: center;
 	}
 	${from.desktop} {
-		grid-row: 1 / 2;
-		align-self: end;
-		justify-self: start;
-		margin-bottom: ${space[3]}px;
-		margin-left: ${pillarLinkWidthTablet * pillars.length + space[2]}px;
-	}
-	${from.leftCol} {
-		margin-left: ${pillarLinkWidthLeftCol * pillars.length + space[2]}px;
-	}
-	${from.wide} {
-		margin-left: ${pillarLinkWidthWide * pillars.length + space[2]}px;
-	}
-`;
-
-const navLinkStyles = css`
-	a {
-		color: ${themePalette('--masthead-nav-link-text')};
-		text-decoration: none;
+		/** TODO - include veggie burger in desktop version of <Pillars /> */
+		display: none;
 	}
 `;
 
 const pillarsNavStyles = css`
-	grid-column: content-start / viewport-end;
```

Comment:
Nitpick for later but I _think_ we do want this at viewport end as the nav links go offscreen on smaller devices, so we don't want the container ending before the gutter